sonar-project.properties

```
@@ -0,0 +1,33 @@
+# Organization and project keys are displayed in the right sidebar of the project homepage
+sonar.organization=owncloud-1
+sonar.projectKey=qnap
+sonar.projectVersion=1.0
+sonar.host.url=https://sonarcloud.io
+
+# =====================================================
+#   Meta-data for the project
+# =====================================================
+
+sonar.links.homepage=https://github.com/owncloud/qnap
+sonar.links.ci=https://drone.owncloud.com/owncloud/qnap/
+sonar.links.scm=https://github.com/owncloud/qnap
+sonar.links.issue=https://github.com/owncloud/qnap/issues
+
+# =====================================================
+#   Properties that will be shared amongst all modules
+# =====================================================
+
+# Just look in these directories for code
+sonar.sources=.
+sonar.inclusions=appinfo/**,lib/**
+
+# Pull Requests
+sonar.pullrequest.provider=GitHub
+sonar.pullrequest.github.repository=owncloud/qnap
+sonar.pullrequest.base=${env.SONAR_PULL_REQUEST_BASE}
+sonar.pullrequest.branch=${env.SONAR_PULL_REQUEST_BRANCH}
+sonar.pullrequest.key=${env.SONAR_PULL_REQUEST_KEY}
+
+# Properties specific to language plugins:
+sonar.php.coverage.reportPaths=results/clover-phpunit-php7.3-sqlite.xml,results/clover-phpunit-php7.3-mariadb10.2.xml,results/clover-phpunit-php7.3-postgres13.3-sqlite.xml,results/clover-phpunit-php7.4-sqlite.xml,results/clover-phpunit-php7.4-mariadb10.2.xml
```

```suggestion
sonar.php.coverage.reportPaths=results/clover-phpunit-php7.3-sqlite.xml,results/clover-phpunit-php7.3-mariadb10.2.xml,results/clover-phpunit-php7.3-postgres13.3-sqlite.xml
```

`.drone.star` has coverage false for PHP 7.4 unit tests pipelines. So the 7.4 coverage files will not exist.

(strangely, Sonar Cloud processing does not complain about `reportPaths` that do not exist)